terraform/application/variables.tf

```diff
@@ -10,6 +10,9 @@ variable "environment" {
 variable "azure_resource_prefix" {
   description = "Standard resource prefix. Usually s189t01 (test) or s189p01 (production)"
 }
+variable "config" {
+  description = "The application environment - should match the tfvars file name e.g. review, production..."
```

Comment:
The application ~~environment~~ configuration - 

(environment is the unique name of the deployed environment, like review-1234)